osf/migrations/0003_sync_with_production.py

```diff
@@ -0,0 +1,162 @@
+# Generated by Django 3.2 on 2022-08-08 15:38
+
+from django.db import migrations, models
+from django.db.utils import ProgrammingError
+from osf.utils.datetime_aware_jsonfield import DateTimeAwareJSONField, DateTimeAwareJSONEncoder
+
+
+class AlterFieldIfNotAlready(migrations.AlterField):
```

Comment:
While 0002 was only supposed to be relevant for local DBs, this should be only for productions. As you can see each field is just altered to match it's definition in the "initial" migration